### DISCUSSION TITLE
Задачи к следующему релизу (25.01)

### DISCUSSION THREAD

**Discussion by alkoleft** [bia-technologies/yaxunit#458] [General]
Предлагаю обсудить состав следующего релиза [25.01](https://github.com/bia-technologies/yaxunit/milestone/13)

На текущий момент состав такой:

Приоритет 1.

- [ ] #425
- [ ] #389
- [ ] #442
- [ ] #382
- [ ] #450
- [ ] #339

Приоритет 2.

- [ ] #393
- [ ] #432
- [ ] #439
- [ ] #441
- [ ] #99
- [ ] #125

Приоритет 3.

- [ ] #91
- [ ] #115
- [ ] #328
- [ ] #394
- [ ] #395
- [ ] #157

**Comment by alexandr-yang** [comment]
#382 уже реализовал, скоро пришлю ПР
